src/Features/Core/Portable/RemoveUnusedMembers/AbstractRemoveUnusedMembersDiagnosticAnalyzer.cs

```diff
@@ -303,6 +303,14 @@ private void OnSymbolEnd(SymbolAnalysisContext symbolEndContext, bool hasInvalid
                                 ? s_removeUnusedMembersRule
                                 : s_removeUnreadMembersRule;
 
+                            // Do not flag write-only properties that are not read.
```

Comment:
this is a 'what' comment, not a 'why'.  Having a comment htat just repeats exactly what hte code is doing isn't that helpful (although **occasionally** it can be).  What's more helpful is to explain *why* this is the right thing to do.

Reply:
💡 In this case, the 'why' would likely be something like this:

> Write-only properties are assumed to have side effects visible through other means than a property getter. 

Since a test covers this case and the test links to an issue, more detail than this shouldn't be necessary.

Reply:
Done, thanks!

Reply:
Thanks!  :)